docs/manuals/man1/pki.1.md

```diff
@@ -0,0 +1,377 @@
+# pki 1 "February 1, 2019" PKI "pki CLI"
+
+## NAME
+
+pki - Command-Line Interface for accessing PKI services.
+
+## SYNOPSIS
+
+**pki** [*CLI-options*] *command* [*command-arguments*]
+
+## DESCRIPTION
+
+The **pki** command provides a command-line interface allowing clients
+to access various services on the PKI server.
+These services include certificates, groups, keys, security domains, and users.
+  
+## CLI OPTIONS
+
+**-c** *NSS-database-password*  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the NSS database password (mutually exclusive to the **-C** option).
+
+**-C** *NSS-database-password-file*  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the file which contains the NSS database password (mutually exclusive to the **-c** option).
+
+**-d** *NSS-database-location*  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the NSS database location (default: *&#8764;/.dogtag/nssdb*).
+
+**-h** *hostname*  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the hostname (default: hostname of the local machine).
+
+**--help**  
+&nbsp;&nbsp;&nbsp;&nbsp;Prints additional help information.
+
+**--ignore-cert-status** *list*  
+&nbsp;&nbsp;&nbsp;&nbsp;Comma-separated list of ignored certificate validity statuses. Valid values are:
+
+- EXPIRED_CERTIFICATE
```

Comment:
So `_` in markdown usually isn't allowed unless it is escaped. I'd just wrap these in backticks since they're something you'd want to see.

Reply:
Per discussion on IRC, this is fine as-is.